avocado/plugins/xunit.py

```diff
@@ -91,10 +91,13 @@ def _create_failure_or_error(self, document, test, element_type,
         system_out.appendChild(system_out_cdata)
         return element, system_out
 
-    def _render(self, result, max_test_log_size):
+    def _render(self, result, max_test_log_size, override_job_name):
```

Comment:
This new argument should be optional, shoudn't? Also, I would suggest name it only job_name: 
```python
def _render(self, result, max_test_log_size, job_name=None):
    ...
    if job_name:
        testsuite.setAttribute('name', job_name)
    else:
        testsuite.setAttribute('name', os.path.basename(os.path.dirname(result.logfile)))
```

Reply:
This is an internal API of a plugin, I don't think it needs to be backward-compatible. Sure, `job_name` should do, I wanted to be maybe a bit too much accurate.